nf_core/pipeline-template/lib/NfcoreSchema.groovy

```diff
@@ -219,7 +219,7 @@ class NfcoreSchema {
                     }
                 }
             }
-            if(rawSchema.keySet().contains('properties') && rawSchema.get('properties').containsKey(ignore_param)) {
+            if(rawSchema.keySet().contains('properties') && rawSchema.get('properties').keySet().contains(ignore_param)) {
```

Comment:
ah right, yeah if nothing else this looks more consistent too 👍🏻 

I guess that we changed the initial call at some point but as we never tested on a pipeline that had ungrouped params we didn't see the failure / didn't update the second call..

Reply:
Yep, I had a feeling this could happen with exactly this part of the code :/